packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -574,6 +575,41 @@ class ConversationService {
     return estimatedPayloadInBytes > EXTERNAL_MESSAGE_THRESHOLD_BYTES;
   }
 
+  public async clearConversation(
+    conversationId: string,
+    timestamp: number | Date = new Date()
+  ): Promise<PayloadBundleOutgoing> {
+    const messageId = new UUID(4).format();
+
+    if (timestamp instanceof Date) {
+      timestamp = timestamp.getTime();
+    }
+
+    const clearedMessage = Cleared.create({
+      clearedTimestamp: timestamp,
+      conversationId,
+    });
+
+    const genericMessage = GenericMessage.create({
+      [GenericMessageType.CLEARED]: clearedMessage,
+      messageId,
+    });
+
+    const {id: selfConversationId} = await this.getSelfConversation();
+
+    await this.sendGenericMessage(this.clientID, selfConversationId, genericMessage);
+
+    return {
+      conversation: conversationId,
+      from: this.apiClient.context!.userId,
+      id: messageId,
+      messageTimer: this.messageTimer.getMessageTimer(conversationId),
```

Comment:
Why would this message have a timer?

Reply:
Good question. It wouldn't.

Reply:
https://github.com/wireapp/wire-web-packages/pull/956